docs/tutorial/01-moveto-raw.html

```diff
@@ -0,0 +1,114 @@
+<!--
+1. Move the `will-move` divs to an new container with a class of `eoc` at the end of the chapter
```

Comment:
Since it is a tutorial... I wonder if we could sprinkle in some questions for the reader and make each step do only one thing.

Example questions to start before describing the steps:

1. What steps would you need to do manually to get the HTML to look like the "Expected Results"?
1. Without being familiar with recipes, how would you write CSS that results in the "Expected Results"? _(useful for gathering info about what the CSS-Refactor format should look like and gets them thinking)_

Example finer-grained steps:

1. create a new container at the end of a chapter
1. give that container a class of `eoc`
1. add a section title to the new container
    1. give it the class `os-title`
    1. give it the value `End of Chapter Collations`
1. move the `will-move` divs to the container
1. convert each `will-move` to a `<section>` (I'm not sure if that was the intention of step 3)

Reply:
After using those, I found it easier to have one file to build upon rather than having to change the file name while baking each time we tried something new. I don't want too much within one "lesson" but one question per file is quite cumbersome when you have to type your path each time (and remember to switch, not to mention loading all the files in the browser)

I rewrote the questions with what you described in mind, their might be more to add. We went over many questions while working on them but I didn't capture all of them here. Maybe @TxsAdamWest and @brittany-johnson can weigh in whether or not that would be useful.

Reply:
I added a bit more to the notes section for 03-passed-raw.html to look more like the instructions for 01-moveto-raw.html & 02-created-nested-content.html .

I like the additions @helenemccarron made to the exercises with your suggestion ,@philschatz  :)